baggers.js

```diff
@@ -15440,6 +15440,92 @@ var data = {
         "twitter": "dedece35",
         "mail": "ddecarvalho@pagesjaunes.fr"
       }
+    },
+    {
+      "since": "2017-06-30",
+      "name": "Yannick DALBIN",
+      "bio": "Développeur passionné par le web depuis tout petit, aujourd'hui co-foundateur d'HashtagTrophy, formateur et co-fondateur de Startup Marseille - premier espace de co-living dédié aux Startups sur Marseille.",
+      "picture": "https://fr.gravatar.com/userimage/27284879/f77fd2b6d51b61605576721be9a8dc47.jpeg?size=200",
+      "websites": [
+        {
+          "name": "Web",
+          "url": "http://hashtagtrophy.com"
+        },
+        {
+          "name": "Web",
+          "url": "https://yannickdalbin.com"
+        },
+        {
+          "name": "Web",
+          "url": "https://startupmarseille.com"
+        },
+        {
+          "name": "Web",
+          "url": "https://youtube.com/yannickdalbin"
+        },
+        {
+          "name": "Web",
+          "url": "https://twitter.com/yannickda"
+        },
+        {
+          "name": "Web",
```

Comment:
You wrong :)

Facebook make lot of work on Pages ! It's become a powerfull tools !

It's so 2017 :)